File: src/main/java/de/maggiwuerze/xdccloader/events/server/ServerDeleteEvent.java
package de.maggiwuerze.xdccloader.events.server;

import de.maggiwuerze.xdccloader.events.EntityDeleteEvent;
import de.maggiwuerze.xdccloader.model.entity.Server;

public class ServerDeleteEvent extends EntityDeleteEvent<Server> {

    public ServerDeleteEvent(Object source, Server server) {
        super(source, server);
    }

}

File: src/main/kotlin/de/maggiwuerze/xdccwebloader/events/server/ServerDeleteEvent.kt
package de.maggiwuerze.xdccwebloader.events.server

import de.maggiwuerze.xdccwebloader.events.EntityDeleteEvent
import de.maggiwuerze.xdccwebloader.model.entity.Server

class ServerDeleteEvent(source: Any, server: Server) : EntityDeleteEvent<Server?>(source, server)